APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+<APE title here>
+================
+
+author: H. Moritz Günther for the finance committee
```

Comment:
I suggest we list all names here @eteq @kelle @aaryapatil @jdswinbank but I did not want to do that until you say it's OK. We did not discuss that explicitly.

Reply:
Ok with me. Thanks for asking!

Reply:
Likewise!

Reply:
Fine with me too!